Managers/LuaMan.cpp

```diff
@@ -600,7 +600,8 @@ int LuaMan::Create() {
             .def("Stop", (bool (SoundContainer:: *)()) &SoundContainer::Stop)
             .def("Stop", (bool (SoundContainer:: *)(int player)) &SoundContainer::Stop)
             .def("Restart", (bool (SoundContainer:: *)()) &SoundContainer::Restart)
-            .def("Restart", (bool (SoundContainer:: *)(int player)) &SoundContainer::Restart),
+            .def("Restart", (bool (SoundContainer:: *)(int player)) &SoundContainer::Restart)
+            .def("FadeOut", (bool (SoundContainer:: *)(int fadeOutTime)) &SoundContainer::FadeOut),
```

Comment:
Don't need to write out the return type and args for this one. Only needed for overload resolution and this has none.
Also you're returning `bool` here but the method is `void`.

Reply:
Oops, yeah, I just copied it from Restart right above lol